lib/3scale_toolbox/commands/update_command/update_service.rb

```diff
@@ -66,8 +68,16 @@ def endpoint_from_url(url)
             uri.to_s
           end
 
+          # system name only included when specified from options
           def target_service_params(source)
-            source.select { |k, v| Commands.service_valid_params.include?(k) && v }
+            target_service = source.select do |k, v|
+              Commands.service_valid_params.include?(k) \
```

Comment:
Wouldn't it be nicer to provider `service_valid_update_params` that does not have the `system_name` ?

Reply:
I was thinking on that too. But `system_name` is needed sometimes, and it is a valid attribute of the service.